tests/_test_main_helpers.py

```
@@ -0,0 +1,475 @@
+    shutdown with the default executor.
+    """
+    loop = asyncio.get_running_loop()
+    bound_call = functools.partial(func, *args, **kwargs)
+    executor = concurrent.futures.ThreadPoolExecutor(max_workers=1)
+    try:
+        return await loop.run_in_executor(executor, bound_call)
+    finally:
+        executor.shutdown(wait=False)
+
+
+def _close_coro_if_possible(coro: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `coro`

(ANN401)